style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.toolbar {
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  font-family: "Inter", Georgia, "Times New Roman", Times, serif;
+  font-weight: 700;
+  font-size: 18px;
+  height: 60px;
+}
+
+.main-class {
+  font-family: "Inter", Georgia, "Times New Roman", Times, serif;
+  height: 500px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #1c1a19;
+}
+
+.logo {
+  padding: 20px;
+  color: #fff;
+  font-style: normal;
+}
+
+.button {
+  display: flex;
+  align-items: flex-start;
+  padding: 25px;
+}
+
+.Hey-there {
+  font-style: normal;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 56px;
+  color: #ff6b00;
+  margin: 20px;
+}
+
+.i-m {
+  font-family: 'Crete Round', serif;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+  margin: 20px;
+}
+
+.description {
+  font-family: 'Crete Round', serif;
```

Comment:
- [ ] Please I highly recommend making changes to the `font-family` of the `.i-m` class from `Crete Round` to `Poppins` per the required Figam design. 😄